053-SentinelAutomatedResponse/Coach/README.md

```diff
@@ -0,0 +1,68 @@
+
+<!-- REPLACE_ME (this section will be removed by the automation script) -->
+# What The Hack - Sentinel Automated Response - Coach Guide
+<!-- REPLACE_ME (this section will be removed by the automation script) -->
+
+## Introduction
+
+<!-- REPLACE_ME (this section will be removed by the automation script) -->
+Welcome to the coach's guide for the Sentinel Automated Response What The Hack. Here you will find links to specific guidance for coaches for each of the challenges.
+<!-- REPLACE_ME (this section will be removed by the automation script) -->
+
+**NOTE:** If you are a Hackathon participant, this is the answer guide. Don't cheat yourself by looking at these during the hack! Go learn something. :)
+
+## Coach's Guides
+
+This hack includes optional [lecture presentations](../Coach/Resources) that introduces key topics associated with each challenge. It is recommended that the host present each short presentation before attendees kick off that challenge.</br>
```

Comment:
This paragraph about the lecture presentations belongs in the Introduction section above just before the note warning students not to cheat.

Also, the link points to the /Coach/Resources folder where you have placed Presentation-01.pptx and Presentation-02.pptx.   You have two PPTX in the /Coach folder that are not linked: "Intro2Sentinel-C2.pptx" and "Intro2Sentinel.pptx".  These appear to be the same files, but with different names.  

A couple of thoughts here: 
1.) Are there specific times when a coach is supposed to use each lecture?  If so, call that out here in the Coach's guide, which a direct link to each file.  Use a GitHub "raw" link when linking directly to a PPTX file.   You can do that by appending "?raw=true" to the end of each link.  For example: "./Coach/Presentation-01.pptx?raw=true"  This will ensure that when a user clicks on the link, it will open the PPT right in the browser, instead of directing them to the GitHub website where they need an extra click to open the file.
2.) The /Coach/Resources/ folder is not needed.  Please remove this folder and keep the final PPTX files in the /Coach folder itself. 
3.) Clean up and remove the PPTX files that are no longer needed.

Reply:
Thanks for cleaning up the extra PPTX files.  The link to the "lecture presentation" is broken.